cmd/ebrelayer/txs/relay.go

```diff
@@ -0,0 +1,86 @@
+package txs
+
+// ------------------------------------------------------------
+//      Relay
+//
+//      Builds and encodes EthBridgeClaim Msgs with the
+//      specified variables, before presenting the unsigned
+//      transaction to validators for optional signing.
+//      Once signed, the data packets are sent as transactions
+//      on the Cosmos Bridge.
+// ------------------------------------------------------------
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtxb "github.com/cosmos/cosmos-sdk/x/auth/client/txbuilder"
+	amino "github.com/tendermint/go-amino"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/utils"
+
+	"github.com/cosmos/peggy/x/ethbridge"
+	"github.com/cosmos/peggy/x/ethbridge/types"
+)
+
+// RelayEvent
+//
+// Applies validator's signature to an EthBridgeClaim message containing information
+// 		about an event on the Ethereum blockchain before sending it to the Bridge
+//		blockchain. For this relay, the chain id (chainID) and codec (cdc) of the
+//		Bridge blockchain are required.
+//
+func RelayEvent(chainId string, cdc *amino.Codec, validatorAddress sdk.ValAddress, moniker string, passphrase string, claim *types.EthBridgeClaim) error {
+
+	var errs []string
+
+	cliCtx := context.NewCLIContext().
+		WithCodec(cdc).
+		WithAccountDecoder(cdc).
+		WithFromAddress(sdk.AccAddress(validatorAddress)).
+		WithFromName(moniker)
+
+	cliCtx.SkipConfirm = true
+
+	txBldr := authtxb.NewTxBuilderFromCLI().
+		WithTxEncoder(utils.GetTxEncoder(cdc)).
+		WithChainID(chainId)
+
+	err := cliCtx.EnsureAccountExistsFromAddr(sdk.AccAddress(claim.ValidatorAddress))
+	if err != nil {
+		errs = append(errs, err.Error())
```

Comment:
Error appending was added to `relay.go` at your request https://github.com/cosmos/peggy/pull/51#discussion_r296021427. I've removed the statements in https://github.com/musnit/peggy/pull/4/commits/4d859fe237d60a2587fbc6472a9139d87ac26f2f, but going forward we'd like to avoid implementing and subsequently removing fixes.